modules/complete-assisted-installer-oci.adoc

```diff
@@ -16,14 +16,14 @@ After you provision {oci-first} resources and upload {product-title} custom mani
 
 . From the link:https://console.redhat.com/[Red Hat Hybrid Cloud Console] web console, go to the *Host discovery* page.
 
-.Under the *Role* column, select either `Control plane node` or `Worker` for each targeted hostname.
+. Under the *Role* column, select either `Control plane node` or `Worker` for each targeted hostname.
```

Comment:
@skrthomas . Added this fix because of this formatting issue :cry: .

[Published AI doc](https://docs.openshift.com/container-platform/4.15/installing/installing_oci/installing-oci-assisted-installer.html#complete-assisted-installer-oci_installing-oci-assisted-installer).